scripts/zones/La_Theine_Plateau/mobs/Bloodtear_Baldurf.lua

```diff
@@ -19,4 +30,28 @@ end;
 
 function onMobDeath(mob, player, isKiller)
     player:addTitle(THE_HORNSPLITTER);
-end;
\ No newline at end of file
+end;
+
+-----------------------------------
+-- onMobDespawn
+-----------------------------------
+
+function onMobDespawn(mob)
+    local mobID = mob:getID();
+    local chanceForLambert = 0;
+
+    if (GetServerVariable("[POP]Lumbering_Lambert") <= os.time(t)) then
+        local chanceForLambert = math.random(1,100);
```

Comment:
local should only be used when the variable is declared. Assigning new values to it just replace it.